### DISCUSSION TITLE
[Mining Issue] My worker suddenly sutting down for no reason. For what reason?

### DISCUSSION THREAD

**Discussion by ktg49292** [filecoin-project/lotus#6278] [Storage Provider] [answered]
**Describe the problem**
Hello! Everyone.
My worker suddenly sutting down for no reason. For what reason?
The logs are stored like this.

**Version**

Daemon:  1.11.0-dev+mainnet+git.72fc16e8a+api1.0.1
Local: lotus-miner version 1.11.0-dev+mainnet+git.72fc16e8a
lotus-worker version 1.11.0-dev+mainnet+git.72fc16e8a

**Setup**

You miner and daemon setup, including what hardware do you use, your environment variable settings, how do you run your miner and worker, do you use GPU and etc.

**Lotus daemon and miner logs**

miner.log

2021-05-17T02:40:50.476+0900    ^[[34mINFO^[[0m blankhost       go-libp2p-blankhost@v0.2.0/blank.go:201 protocol negotiation failed     {"error": "stream reset"}

worker.log

2021-05-17T02:41:09.659+0900    ^[[33mWARN^[[0m main    lotus-seal-worker/main.go:418   Shutting down...
2021-05-17T02:41:09.666+0900    ^[[33mWARN^[[0m main    lotus-seal-worker/main.go:422   Graceful shutdown successful
2021-05-17T02:41:09.698+0900    ^[[33mWARN^[[0m main    lotus-seal-worker/main.go:97    http: Server closed

**Comment by chenjihu** [comment]
I have tried v1.8.0  v1.9.0RC5,  and the issue is still there.   Calibration Net.

**Reply by mrbaiwei** [reply]
Same issue，1.10.0-rc6+calibnet+git.a641a03，1.11.1-dev，random quit
lotus-seal-worker/main.go:418   Shutting down...

**Comment by de-authority** [comment]
Can you try to change your system ulimt value.

**Comment by chenjihu** [comment]
I found the reason:  the hardware (RAM and Motherboard)  incompatibility problem.  The issue is solved after I replaced the RAM with another brand.

**Reply by Araquiel** [reply]
how to diagnost it? to proove that is a ram problem?

**Reply by Araquiel** [reply]
my reason to shuttdown the worker is ssh. run "lotus worker run" with "nohup", but worker dead with ssh session.
fix - run worker from bash script.

**Comment by zrlgit** [comment]
我也遇到同样的问题，版本一样

**Comment by de-authority** [comment, marked as answer]
as far as i know. 100% this kind of question is cause by run lotus with` nohup  lotus-work run &`. and then if you close the terminal. the process will be killed. 
you can type exit to quit. or run with systemd or screen instead!
